docs/_guide/styles.md

```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use unsafeCSS to pass non-literal values using only trusted sources:
```

Comment:
The replacement for `cssLiteral` is just `css`.

Reply:
Done. Thanks for the clarification.